documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+redirect_uri=https%3A%2F%2Fclient%2Eexample%2Ecom%2Fcb&
+client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&
+client_assertion=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzNkJoZFJrcXQzIiwiYXVkIjoiaHR0cHM6Ly9zZXJ2ZXIuZXhhbXBsZS5jb20iLCJzdWIiOiJzNkJoZFJrcXQzIiwiZXhwIjoxMzAwODE5MzgwLCJpYXQiOjEzMDA4MTkwODAsImp0aSI6ImUyMjA5MTNmLTM5ZjItNDEwNi05ZmU0LWI3YmEzNGJkZjJkNSJ9.khzDX5keZQ6EJiJd8upzUMvAZtH8-PypGrWJYeA-Upx1MRDEr54U0V3T2-XZ93PbQE0waMyn25Z-xiDsCMVazYvOEa_Hz5_mE2x1aVZ8m86Ze4lcLld91w0PwnTKqiYy5-vmauruNJCOFXugO1vc3qytA8m8631LhcbrDBHLLS5ieeeA0dXxDTacTPATb1TfM45ahj3Sb0DBA8pefMxuDRkrUAtMporaN4zJ-EenrB0TGlMXklF-w0XQnXezB4_sZmSKlopH4pfSMBKjj_fScP1OlJydKkrbLQzMNHzY6IIgN_xv2Ed-Rf4_I9jdGC8r3BV4YLlQYklKaFfObr86Ew 
+```
+
+### Access Token Response
+
+#### Successful Response
+
+The authorization server issues an access token and optional refresh token and ID Token, and constructs the response by adding the following parameters to the entity-body of the HTTP response with a 200 (OK) status code:
+
+* **access_token**
+
+  REQUIRED. The access token issued by the authorization server.
+
+
+* **token_type**
+
+  REQUIRED. The type of the token issued. It MUST be `bearer`.
+
+
+* **expires_in**
+
+  RECOMMENDED. The lifetime in seconds of the access token.
+
+
+* **refresh_token**
+
+  OPTIONAL, if scope `offline_access` is not included; otherwise, REQUIRED. The refresh token, which can be used to obtain new access tokens using the same authorization.
+
+
+* **scope**
+
+  OPTIONAL, if identical to the scope requested by the client; otherwise, REQUIRED.
+
+
+* **id_token**
+
+  OPTIONAL, if scope `openid` is not included; otherwise, REQUIRED. ID Token value associated with the authenticated session.
+
+
+The following is a non-normative example of a successful Token Response (the ID Token signature in the example can be verified with the key at [Appendix A.1 Authorization Server RSA Key](#appendix-a1-authorization-server-rsa-key)).
+
+```http 
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+```json
+{
+  "access_token": "SlAV32hkKG",
+  "token_type": "Bearer",
+  "refresh_token": "8xLOxBtZp8",
+  "expires_in": 3600,
+  "id_token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjFlOWdkazcifQ.eyJpc3MiOiJodHRwOi8vc2VydmVyLmV4YW1wbGUuY29tIiwic3ViIjoiYWJiZTBiZDYtYTE4Ny00MmRjLThhYTQtYWI2NWU0NGZkZTYzIiwiYXVkIjoiczZCaGRSa3F0MyIsIm5vbmNlIjoibi0wUzZfV3pBMk1qIiwiZXhwIjoxMzExMjgxOTcwLCJpYXQiOjEzMTEyODA5NzB9.opu_go5CTK3o4BesX7-hChKLvr5oOKzkH_j3D_luWCFPbejZP86tWxJDT_HxrTHvvxBgopLcYLoDOHPCY2RHDvEjyuYvejy93sUQf3YA7O58aPzlSweDbpbjARq45tjEbY5W7MHHiBcg2Az9s2UZOHY6Hpnc5cEIz4S6geD25pSiSGzmLic8kiZtP8SjDEp5vNJedOhjfEp1-xVyOGSujd9pz4jWRNnN61mq0dWwVFwD9pqdXWvyLVcndvEzXTgSLMTX98gd1wykbGef9Ud-1an37_7JFdOB5pROAZDmeLQvfCeaKy6GJF5fftA1aSlC1u-vVDuZP4Ip8zZdM5ShiA"
+}
+```
+#### Error Response
+
+The authorization server responds with an HTTP 400 (Bad Request) status code (unless specified otherwise) and includes the following parameters within the response body:
```

Comment:
Why does this profile need to repeat everything that does not change compared to the original standard? I think this profile should list the differences only. There would be no need having the latest change to this line if this profile would not even contain this line. (the "within"-change)

OIDC mostly points to OAuth2 error response. 
This profile could either point to OIDC token error responses https://openid.net/specs/openid-connect-core-1_0.html#TokenErrorResponse
Or just say that this profile does not change, add or remove any error responses.

The same for successful responses. Let this profile name the specific changes Camara needs for security and interoperability improvements. Otherwise we introduce new ambiguities when re-phrasing what intended to stay the same.

Reply:
As per yesterday's WG meeting call (31/01), after asking the WG participants to consider the different opinions presented during the meeting, the way forward was agreed to go with the self-contained format + compromise to properly note any changes related to the reused standard.